src/inmanta/util/dict_path.py

```diff
@@ -656,12 +860,6 @@ def get_element(self, container: object, construct: bool = False) -> object:
         :raises KeyError: if the element is not found or if more than one occurrence was found.
         """
 
-    def get_elements(self, container: object) -> list[object]:
-        try:
-            return [self.get_element(container, False)]
-        except LookupError:
-            return []
-
```

Comment:
Should this still be removed?

Reply:
Not gonna risk it the day before the release (I don't want the full commit to get reverted)